saleor/csv/tests/export/products_data/utils.py

```diff
@@ -1,23 +1,39 @@
+from .....attribute import AttributeInputType
+
+
 def add_product_attribute_data_to_expected_data(data, product, attribute_ids, pk=None):
     for assigned_attribute in product.attributes.all():
         if assigned_attribute:
             header = f"{assigned_attribute.attribute.slug} (product attribute)"
             if str(assigned_attribute.attribute.pk) in attribute_ids:
+                value = (
+                    assigned_attribute.values.first().slug
+                    if assigned_attribute.attribute.input_type
+                    != AttributeInputType.FILE
+                    else "http://mirumee.com/media/"
+                    + assigned_attribute.values.first().file_url
+                )
```

Comment:
Would be good to reformat, it's difficult to read
```suggestion
                if assigned_attribute.attribute.input_type != AttributeInputType.FILE:
                    value = assigned_attribute.values.first().slug
                else:
                    value = "http://mirumee.com/media/" + assigned_attribute.values.first().file_url
```